### ISSUE TITLE
Hoverbox screenreader issues

### ISSUE THREAD

**Issue by robgasc** [ATLAS-Illinois/Illinois-Framework-Theme#1136]
**Describe the bug**
Using NVDA, and the mouse to direct the screenreader, text inside the hover is not read aloud.

**To Reproduce**
1. Visit a page with a hoverbox.
2. Make sure you install and have active NVDA
3. Move the mouse into the hover box, over the title or body.
4. Only image alt text is read aloud.

**Expected behavior**
NVDA can read the text when mouse is over it.

**Additional context**
NVDA does not work with Edge, Chrome or Firefox is best. Read Aloud in edge also has issues parsing the hoverboxes.

**Comment by magjarvis** [comment]
Per feedback from our QA session, we will need to add an arial-label or aria-labeledby to the body in order for the screen reader to read it aloud. The body will also need to be focusable, but I'm not yet sure how to add that to the element. We could possibly use the `<summary>`/`<details>` tags to do this.
